src/platform/common/scriptConstants.ts

```diff
@@ -1,11 +1,9 @@
-import { EXTENSION_ROOT_DIR } from '../constants.node';
-import * as path from '../../platform/vscode-path/path';
+import * as path from '../vscode-path/path';
 
 export * from './constants';
 
 export namespace DataFrameLoading {
-    export const SysPath = path.join(EXTENSION_ROOT_DIR, 'pythonFiles', 'vscode_datascience_helpers', 'dataframes');
-    export const DataFrameSysImport = `import sys\nsys.path.append("${SysPath.replace(/\\/g, '\\\\')}")`;
+    export const SysPath = path.join('pythonFiles', 'vscode_datascience_helpers', 'dataframes');
```

Comment:
Maybe the name here should change to ScriptSubPath? It's not sys path anymore.

Reply:
Not sure why you'd even need this? Couldn't you just have ScriptPath? (Which I think would make sense to rename to ScriptSubPath so that callers have some indication they need to combine it with the extensionUri. 

Personally I'd prefer callers not to have to think about that at all. Like maybe this changes from a namespace to a static class and then there's a function `getScriptPath(extensionContext: IExtensionContext): string` that does the combination.

That should make `debuggerVariables.node.ts` to be able to be turned into just `debuggerVariables.ts` because it wouldn't need anything node specific anymore